.github/workflows/github-ci.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.10.3"]
```

Comment:
Is there something special about python 3.10.3 for this test, or it's just the latest version of 3.10, and we need _a_ version for testing when `cryptodome` is missing?

Reply:
It is not the latest version and there is nothing special about that version. I just needed a way to execute the test suite without pycryptodome. As I assume all python versions will behave the same in that respect, I wanted to run that just once

Reply:
I like using the [matrix.include](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrixinclude) for this as I think it makes the intention clearer of a "special" job, so have something like:

```yaml
    strategy:
      matrix:
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
        use-cryptodome: [""]
        include:
          - python-version: "3.10"
            use-cryptodome: "false"
```

and then test against `${{ matrix.use-cryptodome }}` vs a random python version.

Reply:
Very nice! I didn't know that this was possible. Thanks for sharing!